.styleguidist/styleguidist.sections.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
         },
         {
           name: 'Spacing',
-          content: './.styleguidist/spacing.md',
+          content: './.styleguidist/spacing.insetXld',
```

Comment:
This should be `spacing.md`.

Reply:
Fixed. Was an error with search replace. Good catch!